src/main/java/io/axoniq/axonserver/connector/impl/FlowControlledStream.java

```diff
@@ -127,6 +134,9 @@ protected void markConsumed() {
 
     @Override
     public void beforeStart(ClientCallStreamObserver<OUT> requestStream) {
+        if (refillBatch > 0) {
```

Comment:
Overriding `beforeStart` in FlowControlledStream we activate manual flow control for several calls. This is a plus, not a problem, in my opinion. Just to be aware of that.

Reply:
Completely agree. That was the intent of this change: to put flow control on all the streams received from the server.